engine/engine/content/builtins/scripts/edn.lua

```diff
@@ -6,6 +6,12 @@
 
 local M = {}
 
+local tostring = tostring
+local string = string
+local debug = debug
+local type = type
+local table = table
```

Comment:
localisation of some frequently used functions/tables

Reply:
I like it! Do we want to take it further and also store local copies of string.format and string.gsub?

Reply:
I tried it and it was Slower O_o
it was deep night, maybe I missunderstood something or maybe some kind of LuaJit optimisaitons, not sure, let me try again

Reply:
Huh, that's a surprise then! Perhaps there's some kind of limit on optimization of locals if they are too many?

Reply:
yeah, it's strange, but new tests shows the same result. The diff is minimum but it's a bit worse 🤷‍♂️